doc/how_to_guides/chomp_planner/chomp_planner_tutorial.rst

```diff
@@ -0,0 +1,128 @@
+
+Getting Started
+---------------
+If you haven't already done so, make sure you've completed the steps in :doc:`Getting Started </doc/tutorials/getting_started/getting_started>`.
+
+You should also have gone through the steps in :doc:`Visualization with MoveIt RViz Plugin </doc/tutorials/quickstart_in_rviz/quickstart_in_rviz_tutorial>`
+
+Prerequisites
+--------------
+To use CHOMP with your robot you must have a MoveIt configuration package for your robot. For example, if you have a Panda robot, it's called ``panda_moveit_config`` and can be found :moveit_resources_codedir:`here <panda_moveit_config/>`. These are typically configured using the :doc:`MoveIt Setup Assistant </doc/examples/setup_assistant/setup_assistant_tutorial>`.
+
+Using CHOMP with Your Robot
+---------------------------
+**Note:** If you plan to use the ``panda_moveit_config`` package from the `ros-planning/moveit_resources <https://github.com/ros-planning/moveit_resources/tree/ros2>`_ repository, these steps are already done for you and you can skip this section. Otherwise, to add the configuration for your robot you must:
+
+#. Create a :codedir:`chomp_demo.launch.py<examples/chomp_planner/launch/chomp_demo.launch.py>` file in the launch directory for your MoveIt config package.
+#. Modify all references to Panda in ``chomp_demo.launch.py`` to point to your custom configuration instead
+#. Ensure you have included a  :moveit_resources_codedir:`chomp_planning.yaml <panda_moveit_config/config/chomp_planning.yaml>` file in the config directory of your MoveIt config package.
+#. Open ``chomp_planning.yaml`` in your favorite editor and change ``animate_endeffector_segment: "panda_rightfinger"`` to the appropriate link for your robot. Feel free to modify any parameters you think may batter suit your needs.
+
+Running the Demo
+----------------
+If you have the ``panda_moveit_config`` from the `ros-planning/moveit_resources <https://github.com/ros-planning/moveit_resources/tree/ros2>`_  repository along with ``moveit2_tutorials`` you can run the demo using: ::
+
+  ros2 launch moveit2_tutorials chomp_demo.launch.py rviz_tutorial:=True
```

Comment:
I get this error when launching:

`[ERROR] [launch]: Caught exception in launch (see debug for traceback): Caught exception when trying to load file of format [py]: 'robot_description_planning'`

I think the launch file needs to be tweaked slightly, similar to what I did in this PR:

https://github.com/ros-planning/moveit2_tutorials/pull/520

Reply:
I think this is actually a problem in MoveIt Configs Builder [here](https://github.com/ros-planning/moveit2/blob/71e6d903927116a59b9a142ef7b501f4f6cd1022/moveit_configs_utils/moveit_configs_utils/moveit_configs_builder.py#L129). I've seen this problem elsewhere, and it goes away if you specify `moveit_configs.pilz_cartesian_limits`. I need to look into it a bit more to be sure though.

Reply:
@AndyZe  Can you test again with https://github.com/ros-planning/moveit2/pull/1583